docs/source/platform/schema-validation.md

```diff
@@ -131,15 +131,37 @@ Of course, it’s also possible to leave the field in place indefinitely!
 
 <h2 id="cli">Checking schema changes with the Apollo CLI</h2>
 
-To check and see the difference between the current published schema and a new version, run the following command, substituting the appropriate GraphQL endpoint URL and an API key:
+To check and see the difference between the current published schema and a new version, make sure you have a valid [service configuration](../resources/apollo-config.html#service-config) like so, substituting the appropriate GraphQL endpoint URL and an API key:
+
+```bash
+# .env
+ENGINE_API_KEY=<API_KEY>
+```
 
 > An API key can be obtained from a service's _Settings_ menu within the [Apollo Engine dashboard](https://engine.apollographql.com/).
 
+```javascript
+// apollo.config.js
+module.exports = {
+  service: {
+    endpoint: {
+      url: 'http://localhost:4000/graphql'
+    }
+  }
+};
+```
+
+And run the following command:
+
 ```bash
-apollo service:check --key="<API_KEY>" --endpoint="http://localhost:4000/graphql"
+npx apollo service:check
 ```
 
-> For accuracy, it's best to retrieve the schema from a running GraphQL server (with introspection enabled), though the CLI also reference a local file. See [config options](../platform/apollo-config.html) for more information.
+> Alternatively, you may skip configuration and pass flags to the command, however we recommend [configuring](../resources/apollo-config.html) the project. The configuration file behaves as a source of truth for various Apollo features including IDE support and other CLI commands.
+
+```bash
```

Comment:
https://deploy-preview-327--apollo-docs-index.netlify.com/docs/platform/schema-validation.html#cli gets a bit busy and hard to follow which portion/code is an aside. Can we make this code snippet quoted or place it after the rest of this section? Alternatively we could add a small header before the next paragraph

You can run the docs locally with `npm start` and they auto update thanks to @abernix's additions